src/components/CreateCourse/components/AddedAuthorsToCourse/AddedAuthorsToCourse.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import AddAuthor from '../AddAuthor/AddAuthor';
+
+import { Author } from '../../../../interfaces';
+
+import './addedAuthorsToCourse.css';
+
+type Props = {
+	authors: Author[]; //масив всіх авторів
+	authorsInCourse: string[]; // масив ІД авторів, які додані до курсу
+	onDeleteAuthor: (authorID: string) => void;
+};
+
+function AddingAuthorsToCourse(props: Props) {
+	const isExistAuthorsList =
+		props.authorsInCourse && props.authorsInCourse.length > 0;
+
+	const authorsToRender = props.authors.filter((author) =>
+		props.authorsInCourse.includes(author.id)
+	);
```

Comment:
use `useMemo` for these kinds of variables.

Reply:
done